### ISSUE TITLE
Unable to create new expectation suite using great expectataion version 0.14.1

### ISSUE THREAD

**Issue by namanchawla** [fivetran/great_expectations#4024]
**Description:**  Unable to create new expectation suite with great_expectations library version 0.14.1.  

**Steps To Reproduce**
    Step 1)  Execute  "great_expectations suite new"

    Output of Step 1: 
      How would you like to create your Expectation Suite?
          1. Manually, without interacting with a sample batch of data (default)
          2. Interactively, with a sample batch of data
          3. Automatically, using a profiler
    Step 2)  Select option 2 (Interactively, with a sample batch of data)  

    Output of Step 2: 
       Select data_connector
          1. default_runtime_data_connector_name
          2. default_configured_data_connector_name
          
    Step 3)  Select option 1 (default_runtime_data_connector_name)  
     
    Output of Step 3:
     It is giving error like IndexError: list index out of range

**Expected Behaviour:**
[Unable to create new suite ge version 0.14.1.docx](https://github.com/great-expectations/great_expectations/files/7904213/Unable.to.create.new.suite.ge.version.0.14.1.docx)

    Using the command great_expectations suite new it should allow me to create new expectation suite as observed in earlier version.

**Environment:**
    Operating System: Windows10
    great_expectation version: 0.14.1

**Comment by AFineDayFor** [comment]
Thanks for bringing it to our attention @namanchawla. We'll get back to you.

**Comment by ngenovictor** [comment]
Same here. Following https://docs.greatexpectations.io/docs/tutorials/getting_started/tutorial_create_expectations with my own dataset using option 3 raises an IndexError

Here is the traceback
```shell
great_expectations suite new
Using v3 (Batch Request) API

How would you like to create your Expectation Suite?
    1. Manually, without interacting with a sample batch of data (default)
    2. Interactively, with a sample batch of data
    3. Automatically, using a profiler
: 3

A batch of data is required to edit the suite - let's help you to specify it.

Traceback (most recent call last):
  File "<path-to-env>/py3env/bin/great_expectations", line 8, in <module>
    sys.exit(main())
  File "<path-to-env>/py3env/lib/python3.6/site-packages/great_expectations/cli/cli.py", line 184, in main
    cli()
  File "<path-to-env>/py3env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "<path-to-env>/py3env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "<path-to-env>/py3env/lib/python3.6/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "<path-to-env>/py3env/lib/python3.6/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "<path-to-env>/py3env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "<path-to-env>/py3env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "<path-to-env>/py3env/lib/python3.6/site-packages/click/decorators.py", line 21, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "<path-to-env>/py3env/lib/python3.6/site-packages/great_expectations/cli/suite.py", line 143, in suite_new
    batch_request=batch_request,
  File "<path-to-env>/py3env/lib/python3.6/site-packages/great_expectations/cli/suite.py", line 318, in _suite_new_workflow
    raise e
  File "<path-to-env>/py3env/lib/python3.6/site-packages/great_expectations/cli/suite.py", line 241, in _suite_new_workflow
    additional_batch_request_args=additional_batch_request_args,
  File "<path-to-env>/py3env/lib/python3.6/site-packages/great_expectations/cli/toolkit.py", line 1090, in get_batch_request_using_datasource_name
    additional_batch_request_args=additional_batch_request_args,
  File "<path-to-env>/py3env/lib/python3.6/site-packages/great_expectations/cli/batch_request.py", line 90, in get_batch_request
    msg_prompt_enter_data_asset_name=msg_prompt_enter_data_asset_name,
  File "<path-to-env>/py3env/lib/python3.6/site-packages/great_expectations/cli/batch_request.py", line 233, in _get_data_asset_name_from_data_connector
    available_data_asset_names, msg_prompt_enter_data_asset_name
  File "<path-to-env>/py3env/lib/python3.6/site-packages/great_expectations/cli/batch_request.py", line 257, in _list_available_data_asset_names
    current_page = data_asset_pages[display_idx]
IndexError: list index out of range
```

Using version 0.14.8

Was able to reproduce this on a new python environment with all new installation  of great_expectations. These are versions of dependencies
```
Name: great-expectations
Version: 0.14.8
Summary: Always know what to expect from your data.
Home-page: https://github.com/great-expectations/great_expectations
Author: The Great Expectations Team
Author-email: team@greatexpectations.io
License: Apache-2.0
Location: <path to new env>/env/lib/python3.6/site-packages
Requires: importlib-metadata, tqdm, altair, jsonschema, typing-extensions, scipy, jinja2, mistune, numpy, jsonpatch, pandas, pyparsing, requests, ruamel.yaml, termcolor, pytz, ipywidgets, tzlocal, python-dateutil, Click
Required-by:
altair==4.1.0
appnope==0.1.2
argon2-cffi==21.3.0
argon2-cffi-bindings==21.2.0
async-generator==1.10
attrs==21.4.0
backcall==0.2.0
backports.zoneinfo==0.2.1
black==22.1.0
bleach==4.1.0
certifi==2021.10.8
cffi==1.15.0
charset-normalizer==2.0.12
click==8.0.4
dataclasses==0.8
decorator==5.1.1
defusedxml==0.7.1
entrypoints==0.4
flake8==4.0.1
great-expectations==0.14.8
idna==3.3
importlib-metadata==4.2.0
importlib-resources==5.4.0
ipykernel==5.5.6
ipython==7.16.3
ipython-genutils==0.2.0
ipywidgets==7.6.5
jedi==0.17.2
Jinja2==3.0.3
jsonpatch==1.32
jsonpointer==2.2
jsonschema==4.0.0
jupyter-client==7.1.2
jupyter-core==4.9.2
jupyterlab-pygments==0.1.2
jupyterlab-widgets==1.0.2
MarkupSafe==2.0.1
mccabe==0.6.1
mistune==0.8.4
mypy-extensions==0.4.3
nbclient==0.5.9
nbconvert==6.0.7
nbformat==5.1.3
nest-asyncio==1.5.4
notebook==6.4.8
numpy==1.19.5
packaging==21.3
pandas==1.1.5
pandocfilters==1.5.0
parso==0.7.1
pathspec==0.9.0
pexpect==4.8.0
pickleshare==0.7.5
platformdirs==2.4.0
prometheus-client==0.13.1
prompt-toolkit==3.0.28
ptyprocess==0.7.0
pycodestyle==2.8.0
pycparser==2.21
pydocstyle==6.1.1
pyflakes==2.4.0
Pygments==2.11.2
pyparsing==2.4.7
pyrsistent==0.18.0
python-dateutil==2.8.2
pytz==2021.3
pytz-deprecation-shim==0.1.0.post0
pyzmq==22.3.0
requests==2.27.1
ruamel.yaml==0.17.17
ruamel.yaml.clib==0.2.6
scipy==1.5.4
Send2Trash==1.8.0
six==1.16.0
snowballstemmer==2.2.0
termcolor==1.1.0
terminado==0.13.0
testpath==0.6.0
tomli==1.2.3
toolz==0.11.2
tornado==6.1
tqdm==4.62.3
traitlets==4.3.3
typed-ast==1.5.2
typing-extensions==4.1.1
tzdata==2021.5
tzlocal==4.1
urllib3==1.26.8
wcwidth==0.2.5
webencodings==0.5.1
widgetsnbextension==3.5.2
zipp==3.6.0
```

**Comment by sherrywang31** [comment]
Getting same error following the tutorial using version 0.14.9

**Comment by cdkini** [comment]
Hey all! Apologies for the delayed response.

I'm working to resolve this issue right now - here's the PR: https://github.com/great-expectations/great_expectations/pull/4346

Appreciate your patience 🙇🏽

**Comment by cdkini** [comment]
@namanchawla @ngenovictor @sherrywang31 would you mind pulling the changes from our `develop` branch and trying again?

I've pushed a hotfix and it seems to work locally but I'd like to confirm that it works for you as well.

**Comment by ngenovictor** [comment]
Did `pip install git+https://github.com/great-expectations/great_expectations.git@f519011c6774694369663863ec659a5dcd09a609` from this commit https://github.com/great-expectations/great_expectations/commit/f519011c6774694369663863ec659a5dcd09a609 and a fresh start of the tutorial and everything went fine.

**Comment by cdkini** [comment]
great, thank you for confirming @ngenovictor. I'll keep this issue open for a little bit just to confirm it works with some edge cases but I'm glad to hear it's working now.

**Comment by cdkini** [comment]
I've done some additional testing and think this is probably good to close down - if you do happen to run into another issue, please let us know in Slack or in a new issue and we'll be sure to take a look.

Thanks for your patience and happy coding 🙇🏽
